spec/models/manageiq/providers/kubernetes/container_manager/entity_spec.rb

```diff
@@ -0,0 +1,61 @@
+require "spec_helper"
+
+describe ContainerProject do
```

Comment:
why are all these tests for various classes in a single spec file, instead of being split out into a spec per class/model?